lib/ansible/modules/cloud/azure/azure_rm_autoscale.py

```diff
@@ -550,8 +550,9 @@ def create_rule_instance(params):
                                                                schedule=(RecurrentSchedule(time_zone=p.get('recurrence_timezone'),
                                                                                            days=p.get('recurrence_days'),
                                                                                            hours=p.get('recurrence_hours'),
-                                                                                           minutes=p.get('recurrence_mins')))
-                                                               if p.get('recurrence_frequency') else None)) for p in self.profiles or []]
+                                                                                           minutes=p.get('recurrence_mins'))))
```

Comment:
so this parameter was in totally wrong place right?

Reply:
Here add another conditional check for one case